### PR TITLE
New Tutorial: Start an EC2 Mac instance

### PR DESCRIPTION
**Pull request by cobusbernard**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
Please use this PR going forward, old PR #380. Due to making repo public.

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+Getting access to a macOS-based machine might take time because of additional procurement, installation, and configuration processes. To make it easier, you can choose to start a macOS-based machine on [Amazon EC2](https://aws.amazon.com/ec2/getting-started/).
+
+This series of tutorials takes you through the typical tasks required to start, connect, or configure macOS-based machines in the AWS cloud. Each tutorial is independent from each other, you don't have to follow them all in sequence. Just pick and read the ones that cover the tasks you want to do.
+
+For this series of tutorial, we assume you're somewhat familiar with Amazon EC2. You can quickly jump in by reading [this short tutorial](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html). We will rather focus on the aspects that are specific to Mac on EC2.
+
+This is a 11 parts article about EC2 Mac instances and advanced CLI usage on macOS, including command-line build, test, sign, archive and deploy.
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:28]:
```suggestion
This is a 11 part article about EC2 Mac instances and advanced CLI usage on macOS, including command-line build, test, sign, archive and deploy.
```

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+- This is the first part, you learn how to allocate a Mac mini host and start it.
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:30]:
```suggestion
- This is the first part, where you will learn how to allocate a Mac mini host and start it.
```

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+- When reading [part 2](), you will learn how to remotely connect to the EC2 Mac instance you just started.
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:31]:
```suggestion
- In [part 2](), you will learn how to remotely connect to the EC2 Mac instance you just started.
```

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+![Mac Mini](images/mac-mini-ec2.png)
+
+EC2 Mac instances are Mac mini computers that we racked in our data centers, and connected to the rest of the AWS infrastructure. We use the Mac [Thunderbolt](https://en.wikipedia.org/wiki/Thunderbolt_(interface)) port to connect the machine to [AWS Nitro](https://aws.amazon.com/ec2/nitro/). Nitro is the virtualisation system built by AWS, it allows to access [the VPC network](https://aws.amazon.com/vpc/getting-started/), [the EBS volumes](https://aws.amazon.com/ebs/getting-started/), and other functions. 
+
+There is no virtualisation involved when running an EC2 Mac instance, you have the full Mac mini for you and macOS runs on bare metal. This design decision allows to comply with Apple macOS license that restricts sharing virtualized macOS environments (you might run macOS in a virtual machine on your EC2 instance, but that's a different story). This design decision has two important consequences for you.
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:61]: This design decision allows *you* to comply...

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+To check our [Saving Plans](https://aws.amazon.com/savingsplans/) pricing and options, go to [Saving Plans Compute Pricing page](https://aws.amazon.com/savingsplans/compute-pricing/), then select **EC2 Instance Savings Plans** tab. You can then the length of your plan (**one year or three years**), the payment option (**all upfront, partial upfront, or nothing upfront**), one the **`mac`** instances family, and the AWS Region. Note that, at this time, **OS** must be `linux` and **Tenancy** must be **Dedicated Host**.
+
+![Saving Plans](images/saving-plans.png)
+
+### Acquire a dedicated host
+
+When you want to use an EC2 Mac instance for your development workflow, the first step iswq to reserve a host for your own usage. You can use the [AWS Console](https://eu-central-1.console.aws.amazon.com) or the [command line interface](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to do so. The **Dedicated Hosts** section is available in the EC2 console.
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:73]:
```suggestion
When you want to use an EC2 Mac instance for your development workflow, the first step is to reserve a host for your own usage. You can use the [AWS Console](https://eu-central-1.console.aws.amazon.com) or the [command line interface](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to do so. The **Dedicated Hosts** section is available in the EC2 console.
```

File: tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+}
+```
+
+Keep note of the returned `HostIds` as you will need it to start the EC2 instance on that host. Remember that the billing, and the 24h minimum period of lease time, start at this moment.
+
+## Amazon EC2 Mac instances
+
+Once the host is allocated to your account, you can start or stop the operating system on it. At this point, all the things you know and love from EC2 are also available on macOS. Here is the list of my five favorites features.
+
+- We can create [EBS snapshots](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSSnapshots.html) of my volumes and create [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSSnapshots.html) from these. It means we can capture the state of our disk, snapshot it, and create boot images to start other machines.  This is particularly usefull if you need a common set of tools to be available on all instances and when the installation of these tools takes a long time, for example [Xcode](https://developer.apple.com/xcode/).
```
Comment (bketelsen) [inline tutorials/ec2-mac/01-start-an-ec2-mac-instance/index.md:100]: usefull --> useful

**Review by bketelsen — CHANGES_REQUESTED** [review]
small changes requested. Looks good!

**Comment by sebsto** [comment]
I am unable to edit this one - maybe I need to follow an advanced git course :-) 
So closing this one in favor of 
https://github.com/build-on-aws/content/pull/437

@bketelsen all your suggested changes have been included - thank you.

**Comment by cobusbernard** [comment]
@sebsto : Not your understanding of git :) This is due to making the repo public, it disconnected all private forks, so I recreated the PRs, but due the permissions, only maintainers of this `content` repo can make changes to a PR. Thanks for opening the new one.

**Pull request closed without merging**